change/@fluentui-react-menu-aef37040-8637-4913-8689-7e7ffd018b46.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28162&commitId=8b71ebaf527eebaf74795570072d7e2d7d191316&env=prod&clientType=FLUENTUI)


  <details><summary><b>Menu</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu.Nested Submenus Small Viewport Stacked.nested menu.chromium.png|987| Changed|
|Menu.Nested Submenus Small Viewport Flipped.default.chromium.png|835| Changed|
|Menu.Nested Submenus Small Viewport Stacked.default.chromium.png|835| Changed|
|Menu.Nested Submenus Small Viewport Flipped.nested menu.chromium.png|1444| Changed|

</details>


  <details><summary><b>Menu Converged - MenuItemLinks</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - MenuItemLinks.default.hover menuitemlink.chromium.png|446| Changed|
|Menu Converged - MenuItemLinks.default.chromium.png|450| Changed|

</details>


  <details><summary><b>Menu Converged - basic</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - basic.default.hover menuitem.chromium.png|446| Changed|
|Menu Converged - basic.default.chromium.png|450| Changed|
|Menu Converged - basic.default - RTL.chromium.png|454| Changed|
|Menu Converged - basic.default - RTL.hover menuitem.chromium.png|450| Changed|

</details>


  <details><summary><b>Menu Converged - groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - groups.default - RTL.chromium.png|908| Changed|
|Menu Converged - groups.default - Dark Mode.chromium.png|819| Changed|
|Menu Converged - groups.default.chromium.png|901| Changed|
|Menu Converged - groups.default - High Contrast.chromium.png|1214| Changed|

</details>


  <details><summary><b>Menu Converged - icon slotted content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - icon slotted content.default.hover menuitem.chromium.png|685| Changed|
|Menu Converged - icon slotted content.default - RTL.hover menuitem.chromium.png|696| Changed|
|Menu Converged - icon slotted content.default.chromium.png|695| Changed|
|Menu Converged - icon slotted content.default - RTL.chromium.png|706| Changed|

</details>


  <details><summary><b>Menu Converged - nested submenus</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - nested submenus.default - RTL.all open.chromium.png|1297| Changed|
|Menu Converged - nested submenus.default - RTL.chromium.png|736| Changed|
|Menu Converged - nested submenus.default.all open.chromium.png|1289| Changed|
|Menu Converged - nested submenus.default.chromium.png|731| Changed|

</details>


  <details><summary><b>Menu Converged - secondary content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - secondary content.default - RTL.hover menuitem.chromium.png|609| Changed|
|Menu Converged - secondary content.default - RTL.chromium.png|619| Changed|
|Menu Converged - secondary content.default.hover menuitem.chromium.png|601| Changed|
|Menu Converged - secondary content.default.chromium.png|610| Changed|

</details>


  <details><summary><b>Menu Converged - selection</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection.checkbox - High Contrast.chromium.png|607| Changed|
|Menu Converged - selection.checkbox - Dark Mode.selected.chromium.png|418| Changed|
|Menu Converged - selection.checkbox - Dark Mode.chromium.png|410| Changed|
|Menu Converged - selection.checkbox - High Contrast.selected.chromium.png|582| Changed|
|Menu Converged - selection.checkbox - RTL.selected.chromium.png|485| Changed|
|Menu Converged - selection.checkbox - RTL.chromium.png|454| Changed|
|Menu Converged - selection.checkbox.chromium.png|450| Changed|
|Menu Converged - selection.checkbox.selected.chromium.png|481| Changed|

</details>


  <details><summary><b>Menu Converged - selection groups</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - selection groups.default - RTL.chromium.png|963| Changed|
|Menu Converged - selection groups.default.chromium.png|965| Changed|
|Menu Converged - selection groups.default.selected.chromium.png|996| Changed|
|Menu Converged - selection groups.default - RTL.selected.chromium.png|994| Changed|

</details>


  <details><summary><b>Menu Converged - split item</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - split item.default - RTL.chromium.png|654| Changed|
|Menu Converged - split item.default.submenu open.chromium.png|1208| Changed|
|Menu Converged - split item.default.chromium.png|651| Changed|
|Menu Converged - split item.default - RTL.submenu open.chromium.png|1205| Changed|

</details>


  <details><summary><b>Menu Converged - submenuIndicator slotted content</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu Converged - submenuIndicator slotted content.default.chromium.png|169| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.chromium.png|169| Changed|
|Menu Converged - submenuIndicator slotted content.default.submenus open.chromium.png|970| Changed|
|Menu Converged - submenuIndicator slotted content.default - RTL.submenus open.chromium.png|967| Changed|

</details>


  <details><summary><b>Menu nested within a MenuTrigger</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Menu nested within a MenuTrigger.default.chromium.png|602| Changed|
|Menu nested within a MenuTrigger.default.submenu open.chromium.png|1222| Changed|

</details>


  <details><summary><b>Shadow DOM</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Shadow DOM.Portal.normal.chromium.png|581| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>